torch/utils/cpp_extension.py

```diff
@@ -1572,7 +1597,12 @@ def _run_ninja_build(build_directory: str, verbose: bool, error_prefix: str) ->
         raise RuntimeError(message) from e
 
 
-def _import_module_from_library(module_name, path, is_python_module):
+def _import_module_from_library(module_name, path, is_python_module, is_standalone):
```

Comment:
I am NOT CONVINCED AT ALL that `is_standalone` should be handled as a boolean flag here. No code is shared between the two function implementations, the return types differ depending on the setting of this value... seems like this should have just been two functions.

Reply:
Done.